Jumpstart: add link to dismiss Jumpstart inside modal.

jeherve opened the pull request:
Fixes #10686

#### Changes proposed in this Pull Request:
<!--- Explain what functional changes your PR includes -->

* Add a link to skip Jumpstart inside the modal:

<img width="787" alt="screenshot 2018-12-10 at 20 22 00" src="https://user-images.githubusercontent.com/426388/49756076-dce5d800-fcb9-11e8-958e-4562a01f2200.png">

#### Testing instructions:

* checkout this branch.
* Start from a site with a brand new Jetpack installation.
* Connect your site to WordPress.com
* As you come back to your dashboard, check the modal that pops up.

#### Proposed changelog entry for your changes:

* Connection flow: make it easier to skip recommendations when connecting the plugin to WordPress.com.

jeherve committed:
Jumpstart: add link to dismiss Jumpstart inside modal.

Fixes #10686

oskosk commented on _inc/client/components/jumpstart/index.jsx:
```diff
@@ -23,25 +26,40 @@ class JumpStart extends React.Component {
-			{ this.props.isJumpstarting ? __( 'Activating recommended features…' ) : __( 'Activate recommended features' ) }
-		</Button>;
+		return (
+			<div>
```
The fact that this `<div> ` is rendered inside a `<p>` is generating these warnings:

```
warning.js:33 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>. See JumpStart > p > ... > div.
... > p.
```

jeherve replied:
Thank you. Fixed it by removing the additional `p`, as you suggested. It does not change the look of things.

oskosk commented on _inc/client/components/jumpstart/index.jsx:
```diff
@@ -53,21 +71,19 @@ class JumpStart extends React.Component {
-				<p>
-					{ this.activateButton() }
-				</p>
+				<p>{ this.activateButton() }</p>
```
maybe this wrapping `<p>` is not needed

jeherve replied:
Removed in 83ab35073544903f1606959289368cabf2e8b94c

jeherve committed:
Remove unnecessary paragraph tag

The button is now wrapped in a div.

@see https://github.com/Automattic/jetpack/pull/10920#discussion_r240640277

keoshi reviewed (approved):
Works great, ship it!

oskosk reviewed (approved):
LGTM!
